web/src/context/installerL10n.tsx

```diff
@@ -127,7 +127,13 @@ function languageFromLocale(locale: string): string {
  * @see https://www.rfc-editor.org/info/bcp78
  */
 function languageToLocale(language: string): string {
```

Comment:
please note there is also an inverse `languageFromLocale` function above, which also needs fixing, and testing